cruds_adminlte/templates/HttpResponse/403.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
I think this new template should follow the template patterns, please take a look to `templates/adminlte/example.html` to see how to extend from `templates/adminlte/base.html` and just change the titles and the body.